pkg/testcase/helm.go

```diff
@@ -0,0 +1,83 @@
+package testcase
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/rancher/distros-test-framework/factory"
+	"github.com/rancher/distros-test-framework/pkg/assert"
+	"github.com/rancher/distros-test-framework/shared"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+func TestDeployCertManager(version string) {
+	addRepoCmd := "helm repo add jetstack https://charts.jetstack.io && helm repo update"
```

Comment:
those steps and commands are the same for all os ?

Reply:
Yep always the same!